firefox-ios/Client/Frontend/Theme/ThemedTableViewCells/ThemedTableViewCell.swift

```diff
@@ -10,9 +10,13 @@ class ThemedTableViewCell: UITableViewCell, ReusableCell, ThemeApplicable {
     var viewModel: ThemedTableViewCellViewModel?
     var cellStyle: UITableViewCell.CellStyle
 
+    private var centeredButtonLabelMargin = 15.0
+    lazy var centeredButtonLabel: UILabel = .build()
```

Comment:
I think it would be better to create another custom table view cell (e.g. `ThemedCenteredTableViewCell`) instead of adding functionality to this one. This makes the code more readable and also makes sure we don't accidentally introduce bugs with the existing cell.